api/src/pcapi/routes/backoffice_v3/filters.py

```diff
@@ -429,6 +429,12 @@ def format_offer_validation_sub_rule(sub_rule: offers_models.OfferValidationSubR
     return sub_rule_text
 
 
+def format_offer_validation_rule_list(rules: list[offers_models.OfferValidationRule]) -> str:
+    if rules:
+        return ", ".join(rule.name for rule in rules)
+    return ""
```

Comment:
(style)

```suggestion
    return ", ".join(rule.name for rule in rules)
```

Ça marche aussi avec une liste vide.